### PR TITLE
adds Szechuan sauce

### PR DESCRIPTION
**Pull request by JMWTurner**
szechuan

Has a chance to be included in an exotic garnishes crate.

:cl:
* rscadd: Added Sczechuan sauce. Has a chance to be included in an exotic garnishes crate.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -2935,6 +2935,14 @@
 	nutriment_factor = 2 * REAGENTS_METABOLISM
 	color = "#792300" //rgb: 121, 35, 0
 
+/datum/reagent/szechuan
+	name = "Szechuan"
+	id = SZECHUAN
+	description = "A long forgotten sauce of Earth, recently re-discovered. It's taste is said to be stuff of legends."
```
Comment (Exxion) [inline code/modules/reagents/Chemistry-Reagents.dm]: >It's

File: code/modules/reagents/reagent_containers/food/condiment.dm [hunk 2]
```diff
@@ -185,6 +185,10 @@
 				icon_state = CINNAMON
 			if(GRAVY)
 				icon_state = GRAVY
+			if(SZECHUAN)
+				name = "szechuan sauce"
+				desc = "A long forgotten sauce of Earth, recently re-discovered. It's taste is said to be stuff of legends."
```
Comment (Exxion) [inline code/modules/reagents/reagent_containers/food/condiment.dm]: >It's

File: code/modules/reagents/reagent_containers/food/condiment.dm [hunk 3]
```diff
@@ -369,6 +373,14 @@
 	desc = "Still a bit too small to sail on."
 	volume = 50
 
+/obj/item/weapon/reagent_containers/food/condiment/szechuan
+	name = "szechuan sauce"
+	desc = "A long forgotten sauce of Earth, recently re-discovered. It's taste is said to be stuff of legends."
```
Comment (Exxion) [inline code/modules/reagents/reagent_containers/food/condiment.dm]: OK I would have started a review if I'd expected this many occurrences

**Comment by JMWTurner** [comment]
fixed typo

**Comment by clusterfack** [comment]
The memes jack

**Review by PJB3005 — APPROVED** [review]

**Comment by SonixApache** [comment]
Why just the reagent?

File: code/modules/reagents/reagent_containers/food/condiment.dm [hunk 4]
```diff
@@ -185,6 +185,10 @@
 				icon_state = CINNAMON
 			if(GRAVY)
 				icon_state = GRAVY
+			if(SZECHUAN)
+				name = "szechuan sauce"
+				desc = "A long forgotten sauce of Earth, recently re-discovered. Its taste is said to be stuff of legends."
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/condiment.dm:190]: No label saying it's 97 years old?

Your memes are low effort.

**Comment by salamandooo** [comment]
reddit show

File: code/modules/reagents/reagent_containers/food/condiment.dm (hunk 4 above)
Comment (clusterfack) [inline code/modules/reagents/reagent_containers/food/condiment.dm]: With 9 seasonings

**Comment by kol1th** [comment]
Not so fast Morty, you heard your mom, we've got adventures to go on Morty, just you and me and sometimes your sister and sometimes your mom but never your dad! You wanna know why Morty? Because he CROSSED me! Oh it gets darker, Morty. Welcome to the darkest year of our adventures. First thing that’s different:interrobang:️ No more dad, Morty! He threatened to turn me in to the government so I made him and the government go away! I replaced them both as the defacto patriarch of your family and your universe! Your mom wouldn’t have accepted me if I came home without you and your sister, so now you know the real reason I rescued you - I JUST TOOK OVER THE FAMILY MORTY! And if you tell your mom or sister I said any of this, I’ll deny it. And they’ll take my side because I’m a hero, Morty. And now you’re gonna have to do whatever I say Morty. Forever. And I’ll go out and find some more of that Mulan Szechuan Teriyaki dipping sauce, Morty. Because that’s what this is all about, Morty. That’s my one arm man, I’m not driven by avenging my dead family, Morty. That was FAKE. I’m driven by finding that McNugget Sauce! I want that Mulan McNugget Sauce, Morty! That's my series arc, Morty! If it takes 9 seasons I WANT MY MCNUGGET DIPPING SAUCE! SZECHUAN SAUCE MORTY IT’S GONNA TAKE US ALL THE WAY TO THE END MORTY MORE SEASONS MORTY 9 MORE SEASONS UNTIL I GET THAT DIPPING SZECHUAN SAUCE FOR 97 MORE YEARS MORTY I WANT THAT MCNUGGET SAUCE MORTY

**Comment by Intigracy** [comment]
Season 3's off to a great start.

**Comment by clusterfack** [comment]
Oh sure you don't want attack animations from a PR on april first, but you'll take a Rick and Morty episode?

Hypocrites, hypocrites I say!

**Comment by kol1th** [comment]
I'm afraid of trains

**Comment by TheFriggenSawbones** [comment]
i'm afraid of girls

**Comment by yclatious** [comment]
We already know you are a faggot sawbones, no need to announce it on the git.

**Comment by ComicIronic** [comment]
I'm not in favour.

Not only is this lore-breaking (because you can actually obtain it), but it's also got no actual value.

**Comment by kol1th** [comment]
We already know you are a faggot yclatious, no need to announce it on the git.

**Comment by Kammerjunk** [comment]
Sweet memes are not made of this, but who am I to disagree 🙁

**Comment by Zennyization** [comment]
@ComicIronic I think you need to wake up

**Comment by Exxion** [comment]
>Lore

Just to make sure, everyone knows this was an actual thing, right? Rick and Morty didn't make it up.

**Comment by SonixApache** [comment]
what the fuck is this PR

**Comment by yclatious** [comment]
Memes

**Comment by kol1th** [comment]
@SonixApache reddit

**Comment by Exxion** [comment]
I have no idea

**Comment by AI-Is-Rogue** [comment]
No thanks.

**Comment by Shadowmech88** [comment]
Meme debate aside, it seems like the reagent is 100% literally useless.
It should at least be used in a recipe or something.

File: code/modules/reagents/reagent_containers/food/condiment.dm [hunk 5]
```diff
@@ -377,7 +389,7 @@
 	name = "exotic bottle"
 	desc = "If you can see this label, something is wrong."
 	//~9% chance of anything but special sauce, which is .09 chance
```
Comment (JustSumBody) [inline code/modules/reagents/reagent_containers/food/condiment.dm:391]: It's closer to 8% now

Comment (Zennyization) [reply code/modules/reagents/reagent_containers/food/condiment.dm:391]: seems more like 9% to me...

**Comment by coldcola** [comment]
@Shadowmech88 are you implying things have never been merged with no use, with the intention of a use to come afterwards?

**Comment by Kammerjunk** [comment]
Are you implying a use will come afterwards?

**Comment by salamandooo** [comment]
this isn't even a top quality good meme, it's just cradling snatching reddit's new baby meme that they keep showing off to the world in a desperate attempt to force their shitty show made by justin roliand the faggot who doesn't know how to put beside memes and shows and doesn't know when his time is up and when he should lay down like a good boy and just take it you know

**Comment by BobdaBiscuit** [comment]
\>meme sauce pr has over 30 comments already

God bless.

**Comment by yclatious** [comment]
Autism about reddit shows and sauces, only here on /vg/ station

**Comment by kol1th** [comment]
@salamandooo I NEED THAT MCNUGGET SAUCE MORTY

**Comment by TheFriggenSawbones** [comment]
@salamandooo now don';t get me wrong, rick and morty will never be as good as home movies but I think its a pretty good show you're playing your hand too early and showing an intense hatred towards something that, in reality, isnt bad at all. I think you need to chill back and enjoy cartoons for what they are

**Comment by Zennyization** [comment]
@salamandooo you seem slightly off

**Comment by jknpj** [comment]
https://www.youtube.com/watch?v=FXIWyYtq85E

**Comment by Exxion** [comment]
He's right about one thing though
This meme is neither particularly good nor relevant at all to SS13

**Comment by salamandooo** [comment]
@TheFriggenSawbones i would like it better if morty was a cute girl, but it doesn't even have that reedeming factor

**Comment by coldcola** [comment]
@salamandooo I thought maybe when you'd been here for a few months you'd chill out and become a reasonable member of the community but holy shit do you jump on whatever bandwagon is the new hotness.
Jesus christ stop trying so hard.

**Comment by Exxion** [comment]
Oh dear
Maybe let's stop this pointless hostility right now

**Comment by coldcola** [comment]
@TheFriggenSawbones IT'S SPAGHETTI TIME

**Comment by Zennyization** [comment]
@Exxion I think you need to take a step back

**Comment by kol1th** [comment]
@Exxion square up

**Comment by TheFriggenSawbones** [comment]
@coldcola an undertale parody has more views than the real thing, god has turned his gaze away from us

**Comment by salamandooo** [comment]
@coldcola do not be foolish, I've always disliked rick and morty as it's a meme show made by a meme man.

**Comment by Exxion** [comment]
My steps cannot be more back and my squares not more up

I hit my head earlier

**Comment by TheFriggenSawbones** [comment]
Not so fast Morty :runner::skin-tone-1: you heard your mom :person_with_pouting_face::skin-tone-2: we’ve got adventures :fire: to go on Morty :scream::weary::ok_hand: Just you :facepalm::skin-tone-1:‍♂️:gun: and me :man:‍:microscope::gun: Not so fast Morty you heard your mom :person_with_pouting_face::skin-tone-2: we’ve got adventures :fire: to go on Morty :scream::weary::ok_hand: Just you :facepalm::skin-tone-1:‍♂️:gun: and me :man:‍:microscope::gun: and sometimes your sister :information_desk_person::iphone: and sometimes your mom :woman::skin-tone-2:‍⚕️:racehorse: but :bangbang:️ NEVER :raised_hand: your :triumph: dad! :shrug::skin-tone-1:‍♂️:x::mask: You wanna :see_no_evil: know why Morty? :speak_no_evil::100: Because he CROSSED me :rage::no_good:‍♂️ Oh :dizzy_face: it gets darker Morty :sunglasses::new_moon_with_face: Welcome :alien: to the :sob: darkest :smiling_imp: year of our :man:‍:boy:adventures :pray::tired_face: First 1️⃣ thing that’s :woman:‍:microphone: different :interrobang:️ No more dad Morty :cool: He threatened :eyes::thumbsdown: to turn me in :raising_hand:‍♂️:cop: to the government :sweat::rolling_eyes: so I made him :drooling_face::rat: and the government :poop: go away :middle_finger: I repla:confounded:ced them both :last_quarter_moon_with_face::first_quarter_moon_with_face:as the :smiling_imp: defacto :clown: patriarch :older_man::skin-tone-1: of your family :man:‍:girl:‍:boy:and :astonished: your universe :scream::milky_way::raised_hands: Your mom :racehorse: wouldn’t have :wave: accepted :handshake: me :rolling_eyes: if I came home :house_with_garden: without you :boy::skin-tone-1: and your sister :person_with_blond_hair:‍♀️ so now you know :mortar_board: the real :100: reason :star2: I rescued :helmet_with_cross: you :thumbsup: I JUST :nail_care::skin-tone-2: TOOK OVER :top::dizzy_face: THE FAMILY :man:‍:girl:‍:boy: MORTY :person_frowning::skin-tone-1:‍♂️
And :eyes: if you tell :speaking_head: your mom or sister :woman::skin-tone-2:‍⚕️:information_desk_person: I said any of this :speak_no_evil: I’ll deny it :no_good:‍♂️:triumph: And :smirk: they’ll take my side :two_hearts::relieved::couple: because I’m a :man_dancing: hero :medal::trophy: Morty :joy::rofl: And now :punch: you’re gonna have to do :juggler::skin-tone-1:‍♂️:monkey: whatever I say :stuck_out_tongue_closed_eyes: Morty :disappointed_relieved: Forever :clock1::clock5::clock930::clock10::skull: And I’ll go out :walking::skin-tone-2: and find :telescope: some more :scream_cat: of that :tongue: Mulan :martial_arts_uniform::dolls: Szechuan :dragon::izakaya_lantern:Teriyaki :tired_face::sweat_drops: dipping sauce :fire::ok_hand::pray: Morty :sweat_drops::sweat_drops::sweat_drops::raised_hands: Because that’s :point_right: what this :tada: is all about Morty :poultry_leg: That’s my 1️⃣ one :point_up:️ arm :selfie: man :man_dancing: I’m not :x: driven :oncoming_automobile: by avenging my dead :skull: family :sob::family_mwg: Morty :droplet: That :rofl: was :hear_no_evil: FAKE :sparkles: I’m :man:‍:microscope: driven :dash: by finding :eyes: that :weary: McNugget  :fries: Sauce :sweat_drops: I want :scream: that :flushed: Mulan :dragon::martial_arts_uniform: McNugget :clown: Sauce :eggplant::sweat_drops:Morty :sob: That’s :muscle::skin-tone-2: my series :clapper: arc :rainbow: Morty :crown: If it takes :heart:️ 9 :heart:️ seasons :heart:️ I WANT :tired_face: MY :clap: MCNUGGET :poultry_leg::fries: DIPPING SAUCE :sweat_drops:SZECHUAN :fire: SAUCE MORTY :rage::dizzy_face: IT’S  GONNA TAKE :rocket: US:older_man::skin-tone-1: ALL THE WAY :dizzy: TO :point_right: THE :point_right: END MORTY :checkered_flag: 9️⃣ MORE :moneybag: SEASONS :robot: MORTY :gem: 9️⃣ MORE SEASONS :raised_hands: UNTIL I GET :gift: THAT :heart_eyes: DIPPING :bangbang:️:fire: SZECHUAN :dolls: SAUCE :eggplant::sweat_drops::weary: FOR 9️⃣7️⃣ MORE YEARS :hourglass:️:clock: MORTY :skull: I :clap: WANT :clap: THAT :clap: MCNUGGET :clap: SAUCE :clap: MORTY :clap:

**Comment by coldcola** [comment]
@salamandooo 
>meme show made by a meme man

can you provide some context for that?  was Doc and Mharti referencing meme or something? was justin roiland super popular before R&M?  
you really need to calm yourself down, without justin roiland MST3K might not have returned.

**Comment by FudgePucker** [comment]
Quality coding discussion.

**Comment by salamandooo** [comment]
@coldcola justin roliand ruined gravity falls, ruined alex hirsch, and made alex dump manzi like yesterday's spaghetti. this is the initial cause of my hatred for the man, and what makes him a "meme", which is the word I use to communicate here because it's what gets through. It becomes a meme show when reddit sticks their collective dicks into it, turning an alright show into mediocre garbage as the meme creator becomes relaxed and lowers the quality to shit. Now you're stuck with a garbage show with a creator that get's his dick jacked off by millions at once, and so it'll never get better. AND TO MAKE IT WORSE, some reddit nerd wants to put a reference to the show inside of space station thirteen.

**Comment by yclatious** [comment]
What even is there to possibly discuss, its a good item, its as basic as can be, even I could code one of these.

**Comment by kol1th** [comment]
Gravity falls is a tumblr show

**Comment by coldcola** [comment]
@salamandooo 
>ruined gravity falls

stopped reading there, what are you 12?

**Comment by TheFriggenSawbones** [comment]
invader zim was a great show ruined by the hot topic generation, same with nightmare before christmas

**Comment by yclatious** [comment]
Gravity falls was never even good, cmon now.

**Comment by salamandooo** [comment]
@kol1th It was able to be shared until alex hirsch gave up completely, and then it all spiraled down. Then it became full tumblr show, and it's all the combined fault of alex and justin's inflated egos

@coldcola don't act like you were going to do me a favor by reading, you asked so I provided an answer. if you don't read it it's no sweat off my back

**Comment by Shadowmech88** [comment]
Surely a pull request is the perfect place to discuss the merits of various TV shows and content creators.

**Comment by yclatious** [comment]
Actually, this is probably one of the best places to discuss it. Its kinda like /v/, the best place to discuss anything but actual videogames most of the time.

**Comment by salamandooo** [comment]
You can spam emojis and make shitposts to distract, but the fact is that this item does not deserve to be in the game.

**Comment by Kurfursten** [comment]
Opened: One hour ago
Comments: 60

**Comment by Exxion** [comment]
That's far from a record
Hold on let me find it

**Comment by Intigracy** [comment]
The best part about that scene was the music score choice.

**Comment by Exxion** [comment]
I think #7897 was the fastest accumulation of comments ever, but I'm not sure because GitHub only shows the day when something was posted over a week ago

**Comment by Kammerjunk** [comment]
Just hover over the date stamp smh

**Comment by PJB3005** [comment]
This is some meme that went right over my head didn't it.

**Comment by Kammerjunk** [comment]
It's some cartoon called Rick & Morty, I dunno
People are calling it a Reddit cartoon but I don't really visit Reddit so I wouldn't know 😉

**Comment by Kurfursten** [comment]
The dumb thing about this item, really, is that there will never be an attempt to make any recipe use this sauce -- because it's not part of any recipe. On top of that, the sauce doesn't do anything special so you may as well just sprite special bottle and fill it with regular old nutriment or hotsauce or whatever instead of adding a whole new reagent for memes.

**Comment by salamandooo** [comment]
@PJB3005 It's hardly even a meme. It's one small bit from a shitty forced show that's not even that good, but it's a meme and also aired over samurai jack so people go HAHAHAHA we better love this we better shoehorn it into everything. If you have sense you'll close this PR please

**Comment by gbasood** [comment]
@salamandooo 
\>shitty forced show
w o w
I'm not for this PR either but your hatred for the show is so fucking strong it's hilarious

**Comment by yclatious** [comment]
smh tbh fam bbq

**Comment by kol1th** [comment]
@salamandooo Samurai jack is just a meme show that people think are good due to nostalgia goggles

**Comment by Kammerjunk** [comment]
Joke's on you I never watched the original show

**Comment by salamandooo** [comment]
@gbasood Sorry, it is a little over the top. It is annoying that such a thing might be in the game.
@kol1th It's not as good as people say it is. Still better than the sack of overrated shit that is Rick and Morty though. Samurai Jack is new and after ten or so years has come back, so the hype surrounding it is about alright. There is no excuse except for "reddit", for the popularity and forced memes Justin Roliand's unholy creation has

**Comment by kilozombie** [comment]
why are there so many comments
_it's just some shitty sauce from the 90's_

**Comment by salamandooo** [comment]
@Kammerjunk There are some good episodes, but on the whole it's a lot of walking and staring by Jack. I was never a huge fan, but maybe you'll like it.

**Comment by gbasood** [comment]
you're right clearly the only way a thing you don't like is popular is because REDDIT HIVEMIND ILLUMUNATI made it so

**Comment by kol1th** [comment]
@salamandooo Reddit loves samurai jack too

Wait a second... how do you know so much about reddit?

**Comment by yclatious** [comment]
Memes kilo, atleast it isnt another no fruit pie.

**Comment by kilozombie** [comment]
fuck

**Comment by Kammerjunk** [comment]
I dunno, it just didn't air at the hours I watched TV as a kid
@kol1th How do you? 🤔

**Comment by kol1th** [comment]
🤔

**Comment by TheFriggenSawbones** [comment]
https://www.youtube.com/watch?v=Ejy457WNkuE

**Comment by coldcola** [comment]
the enemy of my enemy is my friend

**Comment by salamandooo** [comment]
@kol1th I'm semi normie and interact with people from real life, the word gets around.

**Comment by Exxion** [comment]
Alright now we're getting somewhere

**Comment by ghost** [comment]
God isn't real and murder isn't bad

**Comment by Exxion** [comment]
Woah fuck

**Comment by kilozombie** [comment]
april fool's

**Comment by Kurfursten** [comment]
>Merged

unsub

**Comment by gbasood** [comment]
WHY
WHY WOULD YOU DO THIS

**Comment by kol1th** [comment]
image

**Comment by JMWTurner** [comment]
Jesus christ what have I done

**Comment by Exxion** [comment]
Mistakes have been made

**Comment by yclatious** [comment]
GET THE PITCHFORKS, WE GOT CODERS TO LYNCH

**Comment by SonixApache** [comment]
absolute madman

**Comment by PJB3005** [comment]
afbeelding

**Comment by JMWTurner** [comment]
It was good while it lasted

**Comment by yclatious** [comment]
L Y N C H
Y
N
C
H

**Comment by JMWTurner** [comment]
This is going places

**Comment by kilozombie** [comment]
to the moon baby

**Comment by salamandooo** [comment]
If this doesn't end in it not being merged I am going to code in the mchicken (best fast food sandwich) at least that's a better meme than a piece of shit sauce

**Comment by JMWTurner** [comment]
why is everyone talking about memes? Since when are references memes?

**Comment by ghost** [comment]
@salamandooo i bet some Sczechuan sauce would go good with that mcchicken

**Comment by yclatious** [comment]
I legitimately always thought we neeeded more chicken recipes so please do salaman.

Also, since forever diety.

**Comment by salamandooo** [comment]
@JMWTurner It becomes a meme because it's from a shitty show instead of a good one, and it's am eme because it's forced as fuck.

**Comment by kilozombie** [comment]
Are you okay salamandooo
What did Doc and Marty do to you?

**Comment by SysadminJeroen** [comment]
https://www.youtube.com/watch?v=MdSH0l9NMek

**Comment by salamandooo** [comment]
@kilozombie HAHAHA WOW HE LICKED HIS BALLS WOW SO FUNNY SO EDGY

**Comment by SangriaMonkey** [comment]
:eyes:

**Comment by kilozombie** [comment]
What are you talking about?
I'm referring to the 1985 smash hit, Back to the Future.

**Comment by salamandooo** [comment]
The hack couldn't even make his own characters, he had to rip them from a great movie

**Comment by yclatious** [comment]
I mean, shit, salameme has a point.

**Comment by Intigracy** [comment]
Yeah, Doc calls his grandson Marty bitch all the time and murders people indiscriminately.

**Comment by FudgePucker** [comment]
@salamandooo Jesus Christ you need to expand your vocabulary or something. 25% of your current one is just meme with about 10000 different meanings.

**Comment by salamandooo** [comment]
@FudgePucker Sorry I will try to be an intellectual like you.

**Comment by Exxion** [comment]
Oh fuck this had a changelog.
Well, looks like people who play the game but don't visit GitHub are going to have a nice joke

**Comment by Kammerjunk** [comment]
Highly memed

**Comment by Exxion** [comment]
I'm going to set my hair on fire

**Comment by ghost** [comment]
SLIPPERY LITTLE BASTARD

**Comment by ghost** [comment]
Hello?

**Comment by salamandooo** [comment]
hi

**Pull request merged**